CONTRIBUTING.md

```diff
@@ -142,6 +142,18 @@ pip install -r requirements.txt  # install dependencies
 pip install -e .  # install the local version of Scribe-Data
 pre-commit install  # install pre-commit hooks
 # pre-commit run --all-files  # lint and fix common problems in the codebase
+```
+ - On Windows run: 
+> 
+```
+ # Install the new version of Scribe-Data and reinstalling new packages
+pip uninstall scribe-data
+pip install .
+```
```

Comment:
This line seems a bit confusing. 😅 I think here should be a short explanation why someone needs to uninstall then reinstall Scribe-data? 

```suggestion
 After running above commands, on Windows, if you face any issue, reinstall Scribe-data by running: 

```
pip uninstall scribe-data
pip install .
```
```

Reply:
Sure it makes sense I will make the changes